pages/common/pickle.md

```diff
@@ -0,0 +1,20 @@
+# pickle
+
+> A PHP extension installer based on Composer.
+> More information: <https://github.com/FriendsOfPHP/pickle>.
+
+- Install a specific PHP extension:
+
+`pickle install {{memcache}}`
+
+- Convert an existing PECL extension configuration to a Pickle configuration file:
+
+`pickle convert {{path/to/directory}}`
+
+- Validate a PECL extension:
+
+`pickle validate {{path/to/directory}}`
+
+- Package a PECL extension for releasing:
```

Comment:
"for release" sounds better to me. But not sure if the other one is objectively correct.

Reply:
`for release` sounds better to me too, so I think I'll go with that :+1: